src/schema/api/EntryType.h

```diff
@@ -0,0 +1,10 @@
+#ifndef LOODB_ENTRYTYPE_H
+#define LOODB_ENTRYTYPE_H
+
+enum EntryType {
+  NULL,
```

Comment:
Would you be able to use `NULL_` instead, since `NULL` is a keyword in C++?

Reply:
Yessir!